uat/src/main/java/com/aws/greengrass/testing/mqtt5/client/grpc/GRPCControlServer.java

```diff
@@ -262,33 +282,38 @@ public void closeMqttConnection(MqttCloseRequest request, StreamObserver<Empty>
             responseObserver.onCompleted();
         }
 
+        /**
+         * Handler of PublishMqtt gRPC call.
+         *
+         * @param request incoming request
+         * @param responseObserver response control
+         */
         @Override
         public void publishMqtt(MqttPublishRequest request, StreamObserver<MqttPublishReply> responseObserver) {
-            Mqtt5Message message = request.getMsg();
 
-            logger.atDebug().log("publishMqtt");
+            Mqtt5Message message = request.getMsg();
 
             int qos = message.getQosValue();
             if (qos < QOS_MIN || qos > QOS_MAX) {
-                logger.atWarn().log("qos is invalid can be in range [0,3] but is {}", qos);
+                logger.atWarn().log("invalid QoS {}, must be in range [{},{}]", qos, QOS_MIN, QOS_MAX);
                 responseObserver.onError(Status.INVALID_ARGUMENT
-                                            .withDescription("qos is invalid can be in range [0,3]")
+                                            .withDescription("invalie QoS, must be in range [0,3]")
```

Comment:
invalid

Reply:
Fixed in 1db6093b6edb3f292620f48e0392d713d1900a5d